package.json

```diff
@@ -1,46 +1,40 @@
 {
-  "name": "gatsby-mdx-blog-starter-project",
-  "description": "Gatsby MDX Blog Starter Project",
-  "version": "1.0.0",
-  "author": "Robin Wieruch <hello@rwieruch.com> (https://www.robinwieruch.de/)",
-  "repository": "https://github.com/rwieruch/gatsby-mdx-starter-project",
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby and Carbon",
+  "version": "0.1.0",
+  "author": "Vince Picone",
   "dependencies": {
-    "@mdx-js/mdx": "^0.15.3",
-    "@mdx-js/tag": "^0.15.0",
-    "core-js": "^2.5.7",
-    "gatsby": "^2.0.7",
-    "gatsby-image": "^2.0.8",
-    "gatsby-link": "^2.0.1",
-    "gatsby-mdx": "^0.1.4",
```

Comment:
I think we want to keep mdx packages - @alisonjoseph ?

Reply:
Well it depends, looks like @vpicone goal here was to create a very un-opinionated starter?

Reply:
@alisonjoseph yeah that was the motivation. The mdx packages come along with the blog starters and could be added easily if someone wanted to go that route with the starter when they implement it.